sdk/identity/identity/src/credentials/clientCertificateCredential.ts

```diff
@@ -9,7 +9,33 @@ import { trace } from "../util/tracing";
 import { MsalFlow } from "../msal/flows";
 import { ClientCertificateCredentialOptions } from "./clientCertificateCredentialOptions";
 
-const logger = credentialLogger("ClientCertificateCredential");
+const credentialName = "ClientCertificateCredential";
+const logger = credentialLogger(credentialName);
+
+/**
+ * Required configuration options for the {@link ClientCertificateCredential}, with either the string contents of a PEM certificate, or the path to a PEM certificate.
```

Comment:
It looks like in the case where both are there (which can happen in various ways, or just in plain JS) we default to using `certificate` over `certificatePath` - can we document this in the comment here?

Reply:
oh what if I throw instead? to make it clearer, rather than to do something possibly unexpected

Reply:
Based on our chat on Teams, yeah throwing is fine :) I have pushed the update.